osmosis/assetlist.json

```diff
@@ -1169,6 +1169,53 @@
         "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/_non-cosmos/ethereum/images/page.png",
         "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/_non-cosmos/ethereum/images/page.svg"
       }
+    },
+    {
+      "description": "AUTISM on Osmosis",
+      "denom_units": [
+        {
+          "denom": "ibc/9DDF52A334F92BC57A9E0D59DFF9984EAC61D2A14E5162605DF601AA58FDFC6D",
+          "exponent": 0,
+          "aliases": [
+            "uautism"
+          ]
+        },
+        {
+          "denom": "autism",
+          "exponent": 6
+        }
+      ],
+      "type_asset": "ics20",
+      "base": "ibc/9DDF52A334F92BC57A9E0D59DFF9984EAC61D2A14E5162605DF601AA58FDFC6D",
+      "name": "Injective AUTISM",
+      "display": "autism",
+      "symbol": "AUTISM",
+      "traces": [
+        {
+          "type": "ibc",
+          "counterparty": {
+            "chain_name": "injective",
+            "base_denom": "factory/inj14lf8xm6fcvlggpa7guxzjqwjmtr24gnvf56hvz/autism",
+            "channel_id": "channel-122"
+          },
+          "chain": {
+            "channel_id": "channel-8",
+            "path": "transfer/channel-8/factory/inj14lf8xm6fcvlggpa7guxzjqwjmtr24gnvf56hvz/autism"
```

Comment:
This path doesn't yield the ibc hash denom above when run through a sha256 function. But it does when using channel 122. Either the channels are backwards or the base hash is wrong

Reply:
yep, channels are backwards

Reply:
thx. box changes have been done.